### PR TITLE
Using the new BBSim in kind-voltha

### PR DESCRIPTION
**Pull request by teone**
Using the new BBSim version

### CODE REVIEW

**Comment by teone** [comment]
This PR will go back to use the `latest` available chart for BBSim, at the moment:
```
chartVversion: 3.0.0
containerVersion: 0.0.1
```
In `minimal-values.yaml` we were tracking branch `voltha-2.1` for BBSim, but that's not there in the new repo so I moved it to `master`. I'm also fine to simply use whatever is the chart default, so that you'll get only released versions of BBSim.

File: onos-files/onos-sadis-sample.json [hunk 1]
```diff
@@ -4,12 +4,12 @@
       "cache": {
         "enabled": false,
         "maxsize": 50,
-        "ttl": "PT0m"
+        "ttl": "PT1m"
```
Comment (dbainbri-ciena) [inline onos-files/onos-sadis-sample.json]: What is the value of using the cache when the entries are in the netcfg? The TTL will really only have an effect when the values are fetched from a remote source.

**Comment by dbainbri-ciena** [comment]
This change means `kind-voltha` only works with the new BBSIM, correct? Are we at that point where no further work or value can be derived from the "old" BBSIM? Are we ready to make the swap?

File: onos-files/onos-sadis-sample.json (hunk 1 above)
Comment (teone) [reply onos-files/onos-sadis-sample.json]: that's a good point, I'm used to have a remote sadis server. I'll fix that.

**Comment by teone** [comment]
yes, this change will move `kind-voltha` to the new BBSim, the only features that are missing in the new one (that will be ported soon) are the ones regarding alarm generation. Do we care for those in voltha-2.x at the moment?

If we do I can rework the patch to add a `WITH_NEW_BBSIM=yes` (mutually exclusive with `WITH_BBSIM=yes`) so that we can swap between the two of them (the image management can be a little tricky as the two versions are using the same values, but I'm not sure we need to override every time what is specified in the chart)

**Comment by davidkbainbridge** [comment]
I wouldn't worry about making a patch that allows a swap between old and new BBSIM. if new BBSIM is ready and the time is now to switch, we should just switch. I think the VOLTHA community needs to be told that this is happening and all sim testing going forward is now to be on bbsim2 and bbsim1, as a repo, should be deprecated.

**Comment by teone** [comment]
Sounds good, I'll send out an email on the VOLTHA mailing list and I'll ask Julie to add a discussion point on the Thursday call

**Pull request merged**